CMakeLists.txt

```diff
@@ -180,25 +180,6 @@ if (protobuf_WITH_ZLIB)
   endif (ZLIB_FOUND)
 endif (protobuf_WITH_ZLIB)
 
```

Comment:
I think the error message you added to this file made a lot of sense.  If someone tries to run this config with <17 we should error quickly and early with a clear error message

Reply:
Added